utils/build-script-impl

```diff
@@ -2012,7 +2012,6 @@ for host in "${ALL_HOSTS[@]}"; do
                     -DCLANG_TOOL_ARCMT_TEST_BUILD=NO
                     -DCLANG_TOOL_C_ARCMT_TEST_BUILD=NO
                     -DCLANG_TOOL_C_INDEX_TEST_BUILD=NO
-                    -DCLANG_TOOL_DRIVER_BUILD=NO
```

Comment:
I think Max is correct. We use the Clang driver to build the runtime, so of course we need to build it. But we *don't* need to build it for the host platform (iOS), so I'm not 100% sure this is correct. (I haven't been here recently.)